lib/ansible/modules/cloud/amazon/ec2_placement_group_facts.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ec2_placement_group_facts
+short_description: List EC2 Placement Group(s) details
+description:
+    - List details of EC2 Placement Group(s).
+version_added: "2.5"
+author: "Brad Macpherson (@iiibrad)"
+options:
+  names:
+    description:
+      - A list of names to filter on. If a listed group does not exist, there
+        will be no corresponding entry in the result; no error will be raised.
+    required: false
+    default: []
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details or the AWS region,
+# see the AWS Guide for details.
+
+# List all placement groups.
+- ec2_placement_group_facts:
+  register: all_ec2_placement_groups
+
+# List two placement groups.
+- ec2_placement_group_facts:
+  names:
+     - my-cluster
+     - my-other-cluster
+  register: specific_ec2_placement_groups
+
```

Comment:
Also can you include an example of getting a specific-named placement group out of the list, like: 

```
- debug: msg="{{ specific_ec2_placement_groups | json_query(\"[?name=='my-cluster']\") }}"
```

Reply:
@ryansb - you don't have to escape json_query that much, as backticks are valid string quotes

- debug: msg="{{ specific_ec2_placement_groups | json_query('[?name==`my-cluster`]') }}"

Reply:
Added